rtsp.go

```diff
@@ -315,28 +383,28 @@ func (rc *rtspCamera) initH265(tracks media.Medias, baseURL *url.URL) (err error
 	}
 
 	// For H.265, handle VPS, SPS, and PPS
-	if format.VPS != nil {
-		rc.rawDecoder.decode(format.VPS)
+	if f.VPS != nil {
+		rc.rawDecoder.decode(f.VPS) // nolint:errcheck
```

Comment:
same here re: error checks, also okay to disregard if this was already the behaviour and one of us will do it in a separate ticket.